wallet/src/repository/wallet/mod.rs

```diff
@@ -806,14 +843,6 @@ where
         )?;
         batch.put(keys::account_utxo_set(account).into_bytes(), utxo_set)?;
         batch.put(keys::account_balance(account).into_bytes(), balance)?;
-        batch.put(
-            keys::account_next_index(account, constants::EXTERNAL_KEYCHAIN),
-            next_external_address_index,
-        )?;
-        batch.put(
-            keys::account_next_index(account, constants::INTERNAL_KEYCHAIN),
-            next_internal_address_index,
-        )?;
```

Comment:
Why can this be removed?

Reply:
Because addresses indices are persisted while generating addresses. No need to do it twice. :P

Reply:
Ok, perfect